specification/eventhub/resource-manager/common/v1/definitions.json

```diff
@@ -83,6 +83,59 @@
           "type": "string"
         }
       }
+    },
+    "systemData": {
```

Comment:
>systemData [](start = 5, length = 10)

Recommend referring to common definitions "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/..." instead of copying it to local common files